terraform/istio/install_istio.sh

```diff
@@ -44,17 +48,16 @@ kubectl create secret generic cacerts -n istio-system \
     --from-file=samples/certs/cert-chain.pem
 
 # cd back into istio/
-cd ../
+cd ${WORD_DIR}
```

Comment:
looks like a typo on WORK_DIR

Reply:
Thanks for catching that - Technically this line is unneeded but it feels safer to come back to our workdirectory after leaving.